modules/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/task/ScheduleTimer.java

```diff
@@ -0,0 +1,94 @@
+        args = {@Argument(name = "onTrigger", type = TypeKind.ANY),
+                @Argument(name = "onError", type = TypeKind.ANY),
+                @Argument(name = "timerScheduler", type = TypeKind.STRUCT, structType = "TimerScheduler")},
+        returnType = {@ReturnType(type = TypeKind.INT), @ReturnType(type = TypeKind.ANY)},
+        isPublic = true
+)
+@BallerinaAnnotation(annotationName = "Description", attributes = {@Attribute(name = "value",
+        value = "Schedules the task service with delay and interval") })
+@BallerinaAnnotation(annotationName = "Param", attributes = {@Attribute(name = "onTrigger",
+        value = "The schedule function as any type") })
+@BallerinaAnnotation(annotationName = "Param", attributes = {@Attribute(name = "onError",
+        value = "The error function as any type") })
+@BallerinaAnnotation(annotationName = "Param", attributes = {@Attribute(name = "timerScheduler",
+        value = "The struct with required attributes") })
+@BallerinaAnnotation(annotationName = "Return", attributes = {@Attribute(name = "int)",
+        value = "The identifier of the task") })
+@BallerinaAnnotation(annotationName = "Return", attributes = {@Attribute(name = "any)",
+        value = "The error which is occurred while scheduling the task") })
+public class ScheduleTimer extends AbstractNativeFunction {
+    private static final Log log = LogFactory.getLog(ScheduleTimer.class.getName());
+
+    public BValue[] execute(Context ctx) {
+        FunctionRefCPEntry onTriggerFunctionRefCPEntry = null;
+        FunctionRefCPEntry onErrorFunctionRefCPEntry = null;
+        if (ctx.getControlStackNew().getCurrentFrame().getRefLocalVars()[0] != null && ctx.getControlStackNew()
+                .getCurrentFrame().getRefLocalVars()[0] instanceof BFunctionPointer) {
+            onTriggerFunctionRefCPEntry = ((BFunctionPointer) getRefArgument(ctx, 0)).value();
+        }
+        if (ctx.getControlStackNew().getCurrentFrame().getRefLocalVars()[1] != null && ctx.getControlStackNew()
+                .getCurrentFrame().getRefLocalVars()[1] instanceof BFunctionPointer) {
+            onErrorFunctionRefCPEntry = ((BFunctionPointer) getRefArgument(ctx, 1)).value();
+        }
+        BValue scheduler = getRefArgument(ctx, 2);
+        long delay = ((BStruct) scheduler).getIntField(0);
+        long interval = ((BStruct) scheduler).getIntField(1);
+        log.info("Request has come to schedule the timer with the INITIAL DELAY: " + delay + " and INTERVAL: "
+                + interval);
+        int taskId = -1;
+        BString error = new BString("Unable to schedule the timer");
+        taskId = TaskUtil.generateTaskId(ctx);
+        if (taskId != -1) {
+            TaskScheduler
+                    .triggerTimer(ctx, taskId, delay, interval, onTriggerFunctionRefCPEntry, onErrorFunctionRefCPEntry);
+            String schedulerError = StringUtils.isNotEmpty((String) ctx.getProperty(Constant.ERROR + "_" + taskId))
```

Comment:
Can't we use `.isEmpty()` method of java String class?
Then we can eliminate the use of extra dependency `org.apache.commons.lang3`